### PR TITLE
Implement card for team overview

### PR DESCRIPTION
**Pull request by ChathurikaThathsarani**
new card

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import { Gallery, Rules } from "./views";
 import { Navbar } from "./components";
+import TeamCard from "./components/TeamCard/TeamCard";
```
Comment (miyurugunarathna) [inline src/App.js:3]: Don’t import directly. Import it through component index. It’s easier to manage look nice.

File: src/components/TeamCard/TeamCard.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export default function TeamCard() {
```
Comment (miyurugunarathna) [inline src/components/TeamCard/TeamCard.jsx:3]: Make it as an arrow function.

**Review by miyurugunarathna — CHANGES_REQUESTED** [review]
Also you can rename TeamView.jsx as index.jsx. When you have a separate folder for it, no need to use name again. And please attach a screenshot of the UI in PR.

**Comment by rusiruavb** [comment]
@ChathurikaThathsarani please resolve the conflicts

**Review by miyurugunarathna — APPROVED** [review]
LGTM

**Pull request merged**